Apps/DiceRoller/DiceRoller/ISDRProjectManager.h

```diff
@@ -12,12 +12,12 @@
 #import "DataManager.h"
 #import "Constants.h"
 #import "Waffle.h"
+#import "ViewController.h"
```

Comment:
I don't think we need `ViewController.h` imported here.  Try removing it and ensuring the app still works.